build_projects/dotnet-cli-build/PrepareTargets.cs

```diff
@@ -184,25 +195,41 @@ public static BuildTargetResult DownloadHostAndSharedFxInstallers(BuildTargetCon
             var sharedFrameworkChannel = DependencyVersions.SharedFrameworkChannel;
             var sharedHostChannel = DependencyVersions.SharedHostChannel;
 
-            var sharedFrameworkInstallerFile = c.BuildContext.Get<string>("SharedFrameworkInstallerFile");
-            var sharedHostInstallerFile = c.BuildContext.Get<string>("SharedHostInstallerFile");
-
-            Mkdirp(Path.GetDirectoryName(sharedFrameworkInstallerFile));
-            Mkdirp(Path.GetDirectoryName(sharedHostInstallerFile));
-
-            AzurePublisher.DownloadFile(
-                AzurePublisher.CalculateInstallerBlob(
-                    sharedFrameworkInstallerFile,
-                    sharedFrameworkChannel,
-                    sharedFrameworkVersion),
-                sharedFrameworkInstallerFile).Wait();
-
-            AzurePublisher.DownloadFile(
-                AzurePublisher.CalculateInstallerBlob(
-                    sharedHostInstallerFile,
-                    sharedHostChannel,
-                    hostVersion),
-                sharedHostInstallerFile).Wait();
+            var sharedFrameworkInstallerDownloadFile = Path.Combine(Dirs.CoreSetupDownload, "sharedFrameworkInstaller");
+            var sharedHostInstallerDownloadFile = Path.Combine(Dirs.CoreSetupDownload, "sharedHostInstaller");
```

Comment:
Same two comments in this method as above.  Putting the Mkdirp calls inside the if checks, and moving the "destinationFile" variables inside as well would make this code easier to read coming from the outside.